mozilla/fx-private-relay
# add FxaTokenAuthentication to REST API auth classes

This PR addresses #MPP-1531.

# New feature description
This change adds a new `FxaTokenAuthentication` backend for the API.

# Screenshot
<img width="1392" alt="image" src="https://user-images.githubusercontent.com/71928/152002820-6a1139d7-acfd-4aba-98da-b9f0a74d337b.png">

# How to test
* Check out this branch of the Relay server and `runserver`.
* Check out [this branch](https://github.com/mozilla/fx-private-relay-add-on/pull/268) of the add-on.
* `web-ext run -s src`
* Open a tab to `about:debugging`, and inspect the Relay add-on so you can see its Console
* Back on the add-on first run page, Sign in
   * [ ] You should be able to complete the OAuth flow with Stage FXA server
   * [ ] On the debugging Console, you should see the sequence shown in the screenshot above, showing that the Add-on was able to authorize its requests to the server with an FXA access token and receive the array of aliases.
